app/models/adapters/active_admin_pundit_adapter.rb

```diff
@@ -0,0 +1,46 @@
+module Adapters
+  class ActiveAdminPunditAdapter < ActiveAdmin::PunditAdapter
+    def get_policy(subject, user, resource)
+      "ActiveAdmin::#{subject}Policy".constantize.new(user, resource)
+    end
+
+    def retrieve_policy(subject)
+      case subject
+      when nil then get_policy(subject, user, resource)
+      when Class then get_policy(subject, user, subject.new)
+      else
+        return Pundit.policy!(user, subject) if active_admin_namespace?(subject)
+
+        get_policy(subject.class, user, subject)
+      end
+    end
+
+    def scope_collection(collection, _action = Auth::READ)
```

Comment:
why did you need to add `scope_collection` but you didn't need to add `authorized?` ?